targets/ar71xx-generic

```diff
@@ -325,6 +325,10 @@ device('tp-link-archer-c60-v1', 'archer-c60-v1', {
 	broken = true, -- OOM with 5GHz enabled in most environments
 })
 
+device('tp-link-archer-c60-v2', 'archer-c60-v2', {
+	broken = false, -- Testing done
```

Comment:
i think we need to set "broken = true" for the same reasons like c25/c58/c60v1

Reply:
I think you're right. I was initially more hopeful, because in OpenWRT Techinfo the C60 is explicitly mentioned with "5GHz not supported" while in V2 there is no issue mentioned. Perhaps one should suggest changes do OpenWRT TOH/TechData as well, but that's another story. ;-)

Anyway: In lived life I am quite happy with my device, the neighbours torture it watching HD IPTV with it... :-D

Reply:
yes, in some cases 64m is enough, but reports about problems in more noisy environments lead to us marking ath10k-64m as broken
maybe newer ath10k versions are better, i don't know, but current gluon master branch is still based on OpenWrt 18.06 and therefore no changes to this rule for now.

Reply:
So we're fine to go? Do I have to do anything? It's my very first GitHub / PR, so excuse the noob question! ;-D

Reply:
"i think we need to set "broken = true" for the same reasons like c25/c58/c60v1"
that's what you have to change, copy the broken-line from c25 or c58 or c60v1 instead of setting "broken = false"

Reply:
I didn't know, that though you typed it in I would have to "double copy" it again. Between bringing children to bed etc. I wasn't very comprehensive in this :D 

Done. Done? :D